src/renderers/WebGLRenderer.js

```diff
@@ -331,6 +331,7 @@ class WebGLRenderer {
 			_this.shadowMap = shadowMap;
 			_this.state = state;
 			_this.info = info;
+			_this.background = background; // allows using renderer.background.dispose() when deep cleaning the scene
```

Comment:
Sorry, I'm not voting doing that since the entire component becomes public and that is not wanted.

Reply:
Like I said before it is not our goal to make everything disposable. The backgrounds resources do not need a clean up.

Reply:
Ok, added a new commit replacing exposed background with disposing method

Reply:
TBH, I'm not in favor of `disposeBackground()`. I would rather hide the internal objects from the statistics. 

There is really no reason for users to care about a few internal geometries or materials. They do not harm the runtime behavior of your app in any means. I'm afraid you are talking this issue too precisely.